src/daemon/daemon.h

```diff
@@ -181,6 +181,7 @@ public slots:
     QFuture<void> image_update_future;
     SettingsHandler* instance_mod_handler;
     std::unordered_map<std::string, std::unordered_map<std::string, MountHandler::UPtr>> mounts;
+    std::unordered_map<std::string, std::vector<std::string>> run_at_boot;
```

Comment:
So basically, the role of run_at_boot is holding the commands need to be executed at boot. I am surprised that adding network interface is the first one which requires this. But anyway, it is a good framework for future as well.

Reply:
Indeed!